greenpeace/planet4-plugin-gutenberg-blocks
# PLANET-4814: Move ENForm to "old"

Ref: https://jira.greenpeace.org/browse/PLANET-4814

In preparation for https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/453 to move to beta, moving the current block to "old" name.

## Test 

ENForm block should work the same
- Check existing blocks in Block usage
- Add a block ENForm to a page, select Goal and Form, check that form displays as expected
